src/ontology/mondo-edit.obo

```diff
@@ -293933,20 +293913,36 @@ property_value: confidence "0.7021739130434781" xsd:double
 
 [Term]
 id: MONDO:0014700
-name: neurodevelopmental disorder-craniofacial dysmorphism-cardiac defect-hip dysplasia syndrome due to a point mutation
+name: Au-Klein syndrome
 subset: ordo_etiological_subtype {source="Orphanet:453504"}
-synonym: "AU-Kline syndrome" RELATED [MONDO:Lexical, OMIM:616580]
-synonym: "Au-Kline syndrome" EXACT [Orphanet:453504]
-synonym: "AUKS" RELATED ABBREVIATION [MONDO:Lexical, OMIM:616580]
-synonym: "neurodevelopmental disorder-craniofacial dysmorphism-cardiac defect-hip dysplasia syndrome due to a point mutation" EXACT CLINGEN_PREFERRED []
+subset: ordo_malformation_syndrome {source="Orphanet:2729"}
+synonym: "Au-Kline syndrome" EXACT [OMIM:616580, Orphanet:453504]
+synonym: "AUKS" EXACT ABBREVIATION [MONDO:Lexical, OMIM:616580]
+synonym: "congenital hydronephrosis with cleft palate, characteristic facies, hypotonia, and intellectual disability" RELATED [GARD:0004064]
+synonym: "congenital hydronephrosis with cleft palate, characteristic facies, hypotonia, and mental retardation" RELATED DEPRECATED [GARD:0004064]
+synonym: "hydronephrosis, congenital, with cleft palate, characteristic facies, hypotonia, and intellectual disability" EXACT [OMIM:604916]
+synonym: "hydronephrosis, congenital, with cleft palate, characteristic facies, hypotonia, and mental retardation" EXACT DEPRECATED [OMIM:604916]
```

Comment:
Why and how (manual change?) was this changed from `RELATED DEPRECATED` to `EXACT DEPRECATED`?

Reply:
yes, I changed this manually. All of the OMIM synonyms should generally be exact. We have a ticket to update this:
https://github.com/monarch-initiative/mondo/issues/2629